test/gateway/index.js

```diff
@@ -40,16 +40,4 @@ describe('HTTP GATEWAY', () => {
     fs.readdirSync(path.join(__dirname, '/spec'))
       .forEach((file) => require('./spec/' + file)(http))
   })
```

Comment:
@ya7ya there are not "gateway spec tests" you probably just have "tests" :)

Reply:
@diasdavid ma bad, I'll fix that,
I'm looking at the [`gateway_test.go`](https://github.com/ipfs/go-ipfs/blob/master/core/corehttp/gateway_test.go) to try to port these tests to `js-ipfs` .

Reply:
awesome! That's perfect @ya7ya :)